tools/symbols_map_generator/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd "$(dirname "$0")"
+rm -rf venv
+python3 -m venv venv
+source venv/bin/activate
+pip install -r ./requirements.txt
+./main.py "$@"
```

Comment:
Think we can get rid of this script, let the user decide how they want to supply it?